bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/inst/OIWFS_Feature.java

```diff
@@ -71,46 +67,34 @@ public OIWFS_Feature() {
     public void reinit(TpeImageWidget iw, TpeImageInfo tii) {
         super.reinit(iw, tii);
 
-        TpeContext ctx = iw.getContext();
+        final TpeContext ctx = iw.getContext();
         if (ctx.instrument().isEmpty()) return;
-        SPInstObsComp inst = ctx.instrument().get();
-
-        // XXX For now, assume instruments have same properties.
-        // XXX If this is not the case, the TPE View menu will need to be updated
-        // XXX whenever the instrument changes, since getProperties() is called
-        // XXX once only the first time the TPE is displayed.
+        final SPInstObsComp inst = ctx.instrument().get();
+
+        // For now, assume instruments have same properties.
+        // If this is not the case, the TPE View menu will need to be updated
+        // whenever the instrument changes, since getProperties() is called
+        // once only the first time the TPE is displayed.
+        // Note that BHROS is simply a filter for GMOS-S, so we use GMOS-S for this.
+        _feat = null;
         if (inst instanceof InstNIRI) {
-            if (_niriFeat == null) {
-                _niriFeat = new NIRI_OIWFS_Feature();
-            }
+            if (_niriFeat == null) _niriFeat = new NIRI_OIWFS_Feature();
             _feat = _niriFeat;
-           } else if (inst instanceof InstGmosCommon) {
-                if (_gmosFeat == null) {
-                    _gmosFeat = new GMOS_OIWFS_Feature();
-                }
-                _feat = _gmosFeat;
-           } else if (inst instanceof InstBHROS) {
-                if (_bhrosFeat == null) {
-                    _bhrosFeat = new BHROS_OIWFS_Feature();
-                }
-                _feat = _bhrosFeat;
+        } else if (inst instanceof InstGmosNorth) {
+            if (_gmosNFeat == null) _gmosNFeat = GmosNorthOIWFSFeature$.MODULE$;
+            _feat = _gmosNFeat;
```

Comment:
For these guys, do we still need fields for these?  Could we not just always `return GmosNorthOIWFSFeature$.MODULE$;`?